client/layout/guided-tours/tours/jetpack-video-hosting-tour/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export const JetpackVideoHostingTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			name="init"
+			target=".jetpack-video-hosting-settings .form-toggle__switch"
+			arrow="top-left"
+			placement="below"
+			style={ {
+				animationDelay: '0.7s',
+				zIndex: 1,
+			} }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>
+						{ translate(
+							'Activate this toggle to use our fast WordPress.com servers to host your videos, ad-free.'
```

Comment:
Flip this toggle to use our WordPress.co, servers to host your videos -- they'll be fast _and_ ad-free.